src/appMain/main.cc

```diff
@@ -142,17 +149,25 @@ int32_t main(int32_t argc, char** argv) {
   if (profile_instance.logs_enabled()) {
     // Logger initialization
     // Redefine for each paticular logger implementation
+#ifdef LOG4CXX_LOGGER
     auto logger = std::unique_ptr<logger::Log4CXXLogger>(
         new logger::Log4CXXLogger("log4cxx.properties"));
+#else   // LOG4CXX_LOGGER
+    auto logger = std::unique_ptr<logger::BoostLogger>(
+        new logger::BoostLogger("boostlogconfig.ini"));
+#endif  // LOG4CXX_LOGGER
+
     logger_impl->Init(std::move(logger));
   }
 #endif
 
   threads::Thread::SetNameForId(threads::Thread::CurrentId(), "SDLCore");
 
+#ifdef LOG4CXX_LOGGER
   if (!utils::appenders_loader.Loaded()) {
     SDL_LOG_ERROR("Appenders plugin not loaded, file logging disabled");
   }
+#endif  // LOG4CXX_LOGGER
```

Comment:
Seems this section causes the build to fail if `ENABLE_LOG=OFF`, maybe move it into the `ENABLE_LOG` ifdef?

Reply:
Resolved in https://github.com/smartdevicelink/sdl_core/pull/3571/commits/770ef067050eb4100ea18d03489fe017025f71f8